plugins/matplotlib/src/DashboardPlugin.tsx

```diff
@@ -37,14 +39,18 @@ export function DashboardPlugin({
         return;
       }
       log.info('Panel opened of type', type);
-      const metadata = { id: widgetId, name, type };
       const config = {
         type: 'react-component' as const,
         component: MatPlotLibPanel.COMPONENT,
         props: {
           localDashboardId: id,
           id: panelId,
-          metadata,
+          metadata: {
+            ...(metadata ?? {}),
```

Comment:
Just set `metadata = {}` in the function params? Or do we explicitly use null from DHE for metadata in which case we would get null?

Looks like Chrome's JS engine doesn't care about spreading null or undefined at least, but I'm sure TS doesn't allow it for more correctness

Reply:
Yea I don't know why I didn't do that, good call.